lib/partition_setup.pm

```diff
@@ -252,18 +252,16 @@ sub addlv {
     send_key 'right' if is_sle('<15');
     send_key 'alt-i' if (is_storage_ng_newui);
     wait_still_screen 2;
-    send_key(
-        is_storage_ng_newui ? $cmd{addlv} : $cmd{addpart});
+    send_key(is_storage_ng_newui() ? $cmd{addlv} : $cmd{addpart});
     if (!is_storage_ng) {
-        send_key 'down' for (0..1);
+        send_key 'down' for (0 .. 1);
         save_screenshot;
     }
     assert_screen 'partition-lv-type';
     send_key 'alt-g';
     wait_still_screen 2;
     type_string $args{name};
-    send_key($args{thinpool} ? 'alt-t' : $args{thinvolume} ? 'alt-i' : 'alt-o'
-    );
+    send_key($args{thinpool} ? 'alt-t' : $args{thinvolume} ? 'alt-i' : 'alt-o');
```

Comment:
I wonder if we actually need something like:
`send_key((($args{thinpool}) ? 'alt-t' : $args{thinvolume}) ? 'alt-i' : 'alt-o');`

Reply:
Just put there what was there before https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/5958/ assuming that used to work. But your's make at least some sense to me.

Reply:
I've tried same on osd, which has perl 18, it works like that, so let's try it out like it is. Again, thanks for the quick reaction!

Reply:
What about this:
```perl
send_key($args{thinpool} ? 'alt-t' : ($args{thinvolume} ? 'alt-i' : 'alt-o'));
```
Reading: if `$args{thinpool}` then 'alt-t' otherwise deal with `($args{thinvolume} ? 'alt-i' : 'alt-o')`. `send_key` then sends what went thru.

No idea about the order of precedence hence the latter `()`.

Reply:
I am always glad to help out when Hyper-V tests fail :).